AppCenterAuth/AppCenterAuthTests/MSAuthTests.m

```diff
@@ -1216,4 +1220,67 @@ - (void)testRefreshNeededWithNilAccountTriggersAnonymous {
   [authTokenContextMock stopMocking];
 }
 
+- (void)testContinuePendingSigninAfterConfigDownload {
+  
+  // If
+  [[MSAuth sharedInstance] setEnabled:YES];
+  id authMock = OCMPartialMock(self.sut);
```

Comment:
We should avoid mocking systems we are testing

Reply:
cc @dtrabo

Reply:
Please review once again, maybe I've missed something.